package.json

```diff
@@ -13,6 +13,7 @@
     "build": "yarn ngb:build && yarn demo:build",
     "test": "yarn check-format && yarn ngb:lint && yarn ngb:test",
     "tdd": "yarn ngb:tdd",
+    "e2e": "yarn ngb:e2e",
```

Comment:
For debugging purpose, it would be fine to have a task in order to launch the e2e app only, without running the tests. i.e. "e2e-app"

Reply:
In the same idea of developping/debugging tests, a good idea would be to be able launch the test without the compilation. This way we can have a console with the incremental compilation, and another one where we will launch 'yarn e2e:dev' (without compilation) when needed.

Reply:
Thanks, I've added:
```
"e2e-app:serve": "ng serve e2e-app" → to serve e2e app
"ngb:e2e-noserve": "ng e2e -c noserve" → to only run protractor tests
```